extensions/guacamole-auth-cas/src/main/resources/guac-manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+
+    "guacamoleVersion" : "0.9.11-incubating",
```

Comment:
Correct at the time this PR was originally opened, but needs to be `0.9.12-incubating` now.